src/sentry/static/sentry/app/views/dashboardsV2/widgetQueries.tsx

```diff
@@ -120,21 +127,67 @@ class WidgetQueries extends React.Component<Props, State> {
   async fetchData() {
     const {selection, api, organization, widget} = this.props;
 
-    this.setState({loading: true, results: []});
+    this.setState({loading: true});
```

Comment:
Do we need to clear `errorMessage` here so previous errors don't persist on subsequent requests?